build_tarballs.jl

```diff
@@ -3,35 +3,48 @@
 using BinaryBuilder
 
 name = "LibVPX"
-version = v"5.0.0"
+version = v"1.8.0"
 
 # Collection of sources required to build LibVPX
 sources = [
-    "https://github.com/webmproject/libvpx/archive/v1.7.0/libvpx-1.7.0.tar.gz" =>
-    "1fec931eb5c94279ad219a5b6e0202358e94a93a90cfb1603578c326abfc1238",
+    "https://github.com/webmproject/libvpx/archive/v1.8.0.tar.gz" =>
+    "86df18c694e1c06cc8f83d2d816e9270747a0ce6abe316e93a4f4095689373f6",
 
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
-cd $WORKSPACE/srcdir
-cd libvpx-1.7.0/
+cd $WORKSPACE/srcdir/libvpx-1.8.0/
 sed -i 's/cp -p/cp/' build/make/Makefile
 mkdir libvpx-build
 cd libvpx-build
 CROSS=$target
 apk add diffutils
+apk add yasm
 ../configure --prefix=$prefix --target=generic-gnu --enable-shared --disable-static
```

Comment:
I'm not sure `--target=generic-gnu` is correct for all platforms :confused: On Windows this generates `.so` libraries